pylib/cqlshlib/copyutil.py

```diff
@@ -1299,9 +1299,9 @@ def __init__(self, inpipe, outpipe, worker_pipes, fname, options):
         self.inpipe = inpipe
         self.outpipe = outpipe
         self.worker_pipes = worker_pipes
-        self.inmsg = None  # must be created after forking on Windows
-        self.outmsg = None  # must be created after forking on Windows
-        self.worker_channels = None  # must be created after forking on Windows
+        self.inmsg = None
```

Comment:
Same here: without Windows' issues, should we instead initialize these at declaration time now?